### DISCUSSION TITLE
select PRESETS

### DISCUSSION THREAD

**Discussion by thejob78** [jmcollin78/versatile_thermostat#685] [Ideas]
when the thermostat manually changes the temperature  or you change the temperature via nodered, selecting a temperature equal to that set in the presets ( frost or eco or comfort or boost ) 
the climate object does not signal to switch to the preset, but the preset (attribute preset_mode) remains on none

**Comment by jmcollin78** [comment]
Hello @thejob78 ,

Yes. If you set a temperature that means, you don't want a preset. What is your point ?

**Comment by thejob78** [comment]
hello
1) I may have moved the temperature by mistake
2) those who use automation, in my case with nodered, having many rooms, when a room requests to start the heating, to optimize consumption, my algorithm also increases the temperature of other rooms (one or two)
so there is a continuous movement of temperatures, I find that at the end of the day all the presets have been lost, it is an aesthetic question, before I used Better Thermostat, it set the presets based on the temperature, it is an aesthetic optimization only. tnk

**Reply by jmcollin78** [reply]
I understand you try to do some self-regulation that is natively available in VTherm. I think you can remove your Nodered automation  and let VTherm do its job.
